cfi_intro.adoc

```diff
@@ -51,39 +51,19 @@ not the conventional link register is a indirect jump.
 The RISC-V control-flow integrity (CFI) extension (Zisslpcfi) builds on these
 conventions and hints. 
 
-The term call is used to refer to a `JAL` or `JALR` instruction (and their
-compressed forms `C.JAL` and `C.JALR`) with `x1` or `x5` as the `rd`. A call
-using `JAL` and `C.JAL` is termed a direct call. A call using `JALR` and
-`C.JALR` is termed an indirect call.
+The term call is used to refer to a `JAL` or `JALR` instruction with a link
+register as destination i.e., `rd != x0`. Conventionally the link register is
+`x1` or `x5`. A call using `JAL` or `C.JAL` is termed a direct call. A call
+using `JALR` or `C.JALR` is termed an indirect call.
```

Comment:
Maybe include the expansion of `c.jalr` to `jalr x1, rs1` (like you do for c.jr below) to make it clear why it's always considered a call?

Reply:
Okay.